store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonReader.java

```diff
@@ -88,6 +90,20 @@ public static CarbonReaderBuilder builder(String tablePath, String tableName) {
     return new CarbonReaderBuilder(tablePath, tableName);
   }
 
+  /**
+   * Return a new {@link CarbonReaderBuilder} instance
+   * Default value of table name is table + time
+   *
+   * @param tablePath table path
+   * @return CarbonReaderBuilder object
+   */
+  public static CarbonReaderBuilder builder(String tablePath) {
+    String time = new SimpleDateFormat("yyyyMMddHHmmssSSS").format(new Date());
+    String uniqueName = "table" + time;
```

Comment:
can you get the substring of `tablePath` and append with time

Reply:
ok, I add a convert method for it.

"./testWriteFiles/" => "table_testWriteFiles_20180531101931973"
"hdfs://testWriteFiles/" =>"table_hdfs_testWriteFiles_20180531102022516"
"s3a://sdk/ => "table_s3a_sdk_20180531102058551"